doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -115,10 +115,11 @@ you will need to rebuild Chapel from source in a different configuration.
       # run the sample program
       ./hello
 
-   At this point, you can choose to continue using ``setchplenv.bash``
-   or use another method to configure Chapel. In particular, you might
-   use a :ref:`chplconfig <readme-chplenv.chplconfig>` file or
-   set environment variables your in dot files or your own script.
+   Note that these environment settings will be lost outside of this terminal session.
+   You can choose to source ``setchplenv.bash`` whenever you want to use relevant commands.
+   If you want to automate this setup, refer to :ref:`chplenv.rst <readme-chplenv.recommended_settings>`
+   for essential variables that should be exported in your dotfiles.
```

Comment:
One might argue that the users who are not familiar with the fact that environment variables disappear when they close their terminal would be unfamiliar with the term "export" and maybe even "dotfiles", too.

I am not necessarily suggesting that we should explain what those are, but this statement here adds another potentially confusing statement for those users this PR intends to help.

Reply:
@e-kayrakli - I agree. I don't know if we should try to explain these terms, but maybe we could just state that the commands from https://chapel-lang.org/docs/master/usingchapel/chplenv.html#recommended-settings can be copied into their `~/.bashrc` (or corresponding rc file for the shell they are using).

Reply:
I like that idea.

Potentially we can link to some external page that explains how `.basrc` et al. are used in setting the environment, but probably that's out of scope of this document and I wouldn't necessarily advocate for that.

There'll always be some missing pieces if you are new to all this stuff and jumping into building Chapel directly without getting some background.

Reply:
@e-kayrakli - how do you feel bout the following wording?

> If you want these environment settings to persist for future terminal sessions, copy the commands from :ref:`chplenv.rst <readme-chplenv.recommended_settings>` into your `~/.bashrc` file.

(it looks like this document assumes the user is using bash, which I'm fine with for now).

Reply:
That sounds good to me.

Reply:
@Rapiz1 - can you make the suggested change?